### PR TITLE
Feature/creating new cover

### PR DESCRIPTION
**Pull request by rachaelcarroll**
## 🚨🤦🏽🚨 *Problem:*
 

- Created the feature of iteration 3 ..using input field data to create a new cover instance and add to the existing arrays.  Also, show this new data on the main page. 

## 🌟💡🌟 *Solution:*

-  By following the steps of iteration 3, first by getting the values of the input fields and assigning the event listener to the button click.  Next, we reassigned the main page element's value to input fields value. Finally, we created a function of pushing these input values into the respective arrays.

## 🤔💬🤔 *Test plan:*

- Our proof is that the make your book button makes the home page reappear with the new instance of the cover that we created via the input fields value.  Also, we console logged every step of the function that was required to perform the tasks given in iteration 3. 

## 👁‍👁‍ *Changes:*
 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
##  💠✅💠 *CheckList:* 
  
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -11,33 +11,34 @@ var homeBtn = document.querySelector('.home-button');
...
+///////////////////VARIABLE FOR INPUT FIELDS////////////////////
+var userCover = document.querySelector('#cover');
+var userTitle = document.querySelector('#title');
+var userDescript1 = document.querySelector('#descriptor1');
+var userDescript2 = document.querySelector('#descriptor2');
+var createCoverBtn = document.querySelector('.create-new-book-button');
 
 /////////// GLOBAL VARIABLES //////////
 var savedCovers = [
-  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
```
Comment (Fakhatova) [inline src/main.js:20]: ohhh i think we gotta put this path back, seems like we forgot to use command +z

Comment (rachaelcarroll) [reply src/main.js:20]: okey dokey!

File: src/main.js [hunk 2]
```diff
@@ -11,33 +11,34 @@ var homeBtn = document.querySelector('.home-button');
...
 
 
-
-
 /////// EVENT LISTENERS👇//////
 
 newRandomCoverBtn.addEventListener('click', showRandomCover)
 makeOwnCoverBtn.addEventListener('click', showForm)
-viewSavedCoverBtn.addEventListener('click', showsavedCover)
+viewSavedCoverBtn.addEventListener('click', showSavedCover)
```
Comment (Fakhatova) [inline src/main.js:33]: Nice catch on camel case change.My bad 🤑

Comment (rachaelcarroll) [reply src/main.js:33]: 🙌

**Pull request merged**